ehitchhiking-ui/src/app/shared/components/menu/menu.module.ts

```diff
@@ -2,21 +2,28 @@ import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
 import {MenuComponent} from './menu.component';
 import {
-	MatButtonModule,
-	MatCheckboxModule,
-	MatSidenavModule,
+  MatButtonModule,
+  MatCheckboxModule, MatDialogModule,
+  MatSidenavModule, MatTableModule,
 } from '@angular/material';
 import {FormsModule} from '@angular/forms';
+import {RatePassangersModalComponent} from '../rate-passangers-modal/rate-passangers-modal.component';
+import {RatingComponent} from '../rating/rating.component';
 
 @NgModule({
-	declarations: [MenuComponent],
+	declarations: [ MenuComponent, RatePassangersModalComponent, RatingComponent ],
 	imports: [
 		CommonModule,
 		MatCheckboxModule,
 		MatSidenavModule,
 		FormsModule,
 		MatButtonModule,
+    MatTableModule,
+    MatDialogModule
 	],
 	exports: [MenuComponent],
+  entryComponents: [
+    RatePassangersModalComponent, // todo: move RatePassangersModalComponent to separate module
```

Comment:
move RatePassangersModalComponent to separate module

Reply:
fixed